src/app/api/auth/lib/nextauth-options.ts

```diff
@@ -157,18 +157,23 @@ export const nextAuthOptions: AuthOptions = {
           'No allowed handles or orgs specified, allowing all users.',
         )
 
+        authLogger.info('User is allowed to sign in:', profile.login)
```

Comment:
iirc the second argument has to be an object

```suggestion
        authLogger.info('User is allowed to sign in:', { profile.login })
```

Reply:
Logger puts string into data field directly in splunk, so change is not needed.